### DISCUSSION TITLE
How to handle missing datapoints

### DISCUSSION THREAD

**Discussion by boredace** [patrykandpatrick/vico#439] [Questions]
Can Vico handle missing data points in a line chart by either not drawing a line over missing periods, or better yet extrapolating value and indicating it by dotted line?

**Comment by patrickmichalik** [comment]
Hello! To create a break in a line, you can use multiple series linked to the same `LineSpec`. Dashed and dotted lines and the ability to change a single line’s style depending on the _x_ value are planned additions. Depending on the desired result, you can implement interpolation by using `DefaultPointConnector` and setting `cubicStrength` to zero or by adding interpolated points to your data set. The latter approach can also be used for extrapolation.

**Reply by emkayDauda** [reply]
Hi @patrickmichalik, can you elaborate on `DefaultPointConnector` please? The desired result for my use case is that the graph continues naturally (curving up if the next value is higher)

**Reply by Gowsky** [reply]
Hello, @emkayDauda. Please be more precise. "Continues naturally" is very vague. You are referring to interpolation, correct? What kind of curve do you need exactly? Is it needed between every pair of points or just some? How does it compare to the default behavior?

**Reply by emkayDauda** [reply]
Thanks, I've clarified on a new question here: https://github.com/patrykandpatrick/vico/discussions/965

**Reply by Gowsky** [reply]
Hello. The dashed lines support was added in [Vico `2.0.0-beta.4`](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0-beta.4).
